README.md

```diff
@@ -32,11 +32,18 @@ Still not merged:
    - For Ubuntu: `sudo apt install build-essential linux-headers-generic`
    - For Fedora: `sudo dnf install kernel-devel`
 2. Clone this repository and cd to it
+
+##### Either: For older kernels which do not utilize DKMS (verify that dkms is not available eg. with `which dkms`)
```

Comment:
`older-kernel-patch` is made because from this version kernel started to use different API for some operations, not for DKMS.

> For older kernels which do not utilize DKMS

This could be simplified to something like "check if your distro support DKMS"